### DISCUSSION TITLE
how to zoom chart map with mousewheel ?

### DISCUSSION THREAD

**Discussion by luongnv275** [elimintz/justpy#616] [Q&A]

**Comment by WolfgangFahl** [comment]
As per https://justpy.io/#if-you-have-questions-suggestions-or-would-like-to-get-in-touch i have converted this to a discussion item. 
_If you have a question about JustPy you are welcome to check [stackoverflow questions tagged justpy](https://stackoverflow.com/questions/tagged/justpy) and Especially questions which involve details of upstream frameworks such as [ag-grid](https://stackoverflow.com/questions/tagged/ag-grid), [highcharts](https://www.highcharts.com/), [pandas](https://pandas.pydata.org/), [plotly](https://plotly.com/python/), [quasar](https://quasar.dev/), [tailwind](https://tailwindcss.com/) and others are best ask involving the larger [stackoverflow](https://stackoverflow.com/) community and tagging your question with both "justpy" and the tag of the specific library your are asking a question for such as "ag-grid", "highcharts","quasar", "pandas" and others._

Unforunately your question is very unspecific. You are right that neither "zoom" nor "chart map" nor "mousewheel" give any hit in the search of https://justpy.io/ even a stackoverflow search https://stackoverflow.com/search?q=zoom+chart+map+mousewheel does not give any results.

Still i will try to guess what you are trying to find out.
"chartmap" seems to refer to "highmap". 

Searching the web for zoom highmap there are quite a few answers. One is specifically about zoom by mouse scroll:
https://stackoverflow.com/questions/36490209/highmaps-how-to-disable-map-zoom-by-mouse-scroll

You might want to get yourself a stackoverflow account and upvote the question (5) and answer (11). 

Now the specific setting https://api.highcharts.com/highmaps/mapNavigation.enableMouseWheelZoom mentioned there is something that is worthwile discussing in the context of justpy. Justpy only **transports** settings by "_remote DOM rewriting_". It's a valid question how to do this - please repeat this detailed question with some code example as per the stackoverflow Q&A guidelines on stackoverflow and i'll happily answer you there.

**Comment by WolfgangFahl** [comment]
http://jsfiddle.net/Frank_Lou/wdu9axoL/23/ has an example fiddle where you can try things out with javascript before trying the python remote rewriting.
See also https://github.com/WolfgangFahl/pyJustpyWidgets/blob/main/jpdemo/examples/highchartmap_demo.py  and #452 and #315
